src/channel/rpc_0_6.ts

```diff
@@ -119,16 +154,35 @@ export class RpcChannel {
     }
   }
 
+  /**
+   * Retrieves the chain ID from the Starknet endpoint.
+   * If the chain ID has already been fetched, it returns the cached value.
+   *
+   * @returns - A promise that resolves to the chain ID.
+   */
   public async getChainId() {
     this.chainId ??= (await this.fetchEndpoint('starknet_chainId')) as StarknetChainId;
     return this.chainId;
   }
 
+  /**
+   * Retrieves the spec version of the Starknet chain.
+   *
+   * @returns - A Promise that resolves with the spec version of the Starknet chain.
+   */
   public async getSpecVersion() {
     this.specVersion ??= (await this.fetchEndpoint('starknet_specVersion')) as StarknetChainId;
     return this.specVersion;
   }
 
+  /**
+   * Retrieves the nonce for a given contract address and block identifier.
```

Comment:
For all of these descriptions in the rpc files, it's possible to go through the spec file and compare the summary!
https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json